[IT] Updated `Community Contributors` (#8741)
ppy/osu-wiki: wiki/People/Community_Contributors/it.md

# Collaboratori della Comunità

![Badge dei Collaboratori](/wiki/shared/contributor.jpg "Badge dei Collaboratori")

I **Collaboratori della Comunità** sono le persone che hanno preso l'iniziativa di provvedere servizi alla comunità di osu! a caro prezzo per loro - strumenti e funzionalità che molti di noi danno per scontato ora. Questi utenti ricevono un badge per indicare che hanno fatto qualcosa degno di nota per l'intera comunità.

> Non devono essere confusi con gli [osu! Alumni](/wiki/People/The_Team/osu!_Alumni) che erano membri del team di osu! che si sono ritirati, Il badge per i Collaboratori della Comunità sono dati a liberi professionisti che sono andati al di là del proprio dovere per migliorare osu! e l'ambiente della comunità.

Reviewer: Is `liberi professionisti` actually the word that we are looking for?

Gli utenti attuali che hanno questo badge sono inseriti in questa tabella:

## 2013

### Agosto

*Per il thread sul forum, vedi: [osu! community contributors](https://osu.ppy.sh/community/forums/topics/147919)*

| Utente | Contributi |
| :-- | :-- |
| ::{ flag=GB }:: [Darkimmortal](https://osu.ppy.sh/users/10886) | Ha sviluppato e mantenuto attivo il [servizio osu!record](https://osu.ppy.sh/community/forums/topics/108092) (da .osr a Video su YouTube.) |
| ::{ flag=DE }:: [nanashiRei](https://osu.ppy.sh/users/807630) | Hosta e mantiene uno dei più duraturi [mirror di beatmap](http://osu.yas-online.net/). |
| ::{ flag=US }:: [RBRat3](https://osu.ppy.sh/users/307202) | Numerosi (incalcolabili) contributi grafici al client di gioco e tutto il resto. |
| ::{ flag=CN }:: [Ballance](https://osu.ppy.sh/users/165946) | Grafica degli [Obiettivi](/wiki/Medals). |

> | ::{ flag=US }:: [akrolsmir](https://osu.ppy.sh/users/576800) | Sviluppa e mantiene [AIBat](https://osu.ppy.sh/community/forums/topics/55305), il più popolare strumento di modding delle beatmap di terza parte. |

Reviewer: ```suggestion
| ::{ flag=US }:: [akrolsmir](https://osu.ppy.sh/users/576800) | Sviluppa e mantiene [AIBat](https://osu.ppy.sh/community/forums/topics/55305), il più popolare strumento di modding delle beatmap di terze parti. |
```

> | ::{ flag=NL }:: [statementreply](https://osu.ppy.sh/users/126198) | Performance straordinaria nei bugfix o test delle funzionalità e ha programmato numerose utilities per i mapper e modders. |

Reviewer: ```suggestion
| ::{ flag=NL }:: [statementreply](https://osu.ppy.sh/users/126198) | Ha contribuito a risolvere numerosissimi bug e a provare delle funzionalità. Inoltre, ha creato numerosi strumenti per mapper e modders. |
```

> | ::{ flag=DE }:: [Loctav](https://osu.ppy.sh/users/71366) | Ha organizzato molti tornei ufficiali di maggiore importanza (incluso l'OWC). |

Reviewer: ```suggestion
| ::{ flag=DE }:: [Loctav](https://osu.ppy.sh/users/71366) | Ha organizzato molti tornei ufficiali di grande rilevanza (incluso l'OWC). |
```

| ::{ flag=US }:: [Blazevoir](https://osu.ppy.sh/users/120265) | Folle record di moderazione (equivalente a circa 6 membri dello staff) e ha gestito e moderato manualmente #osu durante le ore di punta (Più di 9000 utenti). |

> | ::{ flag=PL }:: [Piotrekol](https://osu.ppy.sh/users/304520) | Sviluppa e mantiene [osu!stats](http://osustats.ppy.sh/) e un numero imprecisato di utilities utili per fare beatmaps, moddare e giocare in generale. |

Reviewer: ```suggestion
| ::{ flag=PL }:: [Piotrekol](https://osu.ppy.sh/users/304520) | Sviluppa e mantiene [osu!stats](http://osustats.ppy.sh/) e un numero imprecisato di strumenti per creare, moddare e giocare beatmaps in generale. |
```

> | ::{ flag=NO }:: [MillhioreF](https://osu.ppy.sh/users/941094) | Performance straordinaria nei bugfix o test delle funzionalità e nel gestire le richieste di supporto dagli utenti. |

Reviewer: ```suggestion
| ::{ flag=NO }:: [MillhioreF](https://osu.ppy.sh/users/941094) | Ha contribuito a risolvere numerosissimi bug e a provare delle funzionalità. Inoltre, ha gestito numerose richieste di supporto da parte degli utenti. |
```

> | ::{ flag=US }:: [DeathxShinigami](https://osu.ppy.sh/users/49516) | Più di 3 anni di lavoro silenzioso e instancabile nel creare [pacchetti beatmaps](https://osu.ppy.sh/p/packlist) e diversi anni nel gestire le chart. |

Reviewer: ```suggestion
| ::{ flag=US }:: [DeathxShinigami](https://osu.ppy.sh/users/49516) | Più di 3 anni di duro lavoro silenzioso per creare [pacchetti beatmaps](https://osu.ppy.sh/p/packlist) e gestire i grafici |
```

> | ::{ flag=US }:: [LuigiHann](https://osu.ppy.sh/users/1079) | Anni su anni di servizio, incluso il creare la skin per le beatmap originale, creare [storyboard](/wiki/Storyboard) avanzati come un'opera d'arte e ha motivato il team. |

Reviewer: ```suggestion
| ::{ flag=US }:: [LuigiHann](https://osu.ppy.sh/users/1079) | Anni ed anni di servizio. In particolare, ha creato skin per le beatmap originali, creato [storyboard](/wiki/Storyboard) avanzati, comparabili  a vere opere d'arte e ha motivato il team. |
```

| ::{ flag=MX }:: [Repflez](https://osu.ppy.sh/users/201392) | Straordinario Contributo alla osu! wiki. |
| ::{ flag=MY }:: [RaikireHiuduo](https://osu.ppy.sh/users/1570014) | Straordinario Contributo alla osu! wiki. |
| ::{ flag=DE }:: [givenameplz](https://osu.ppy.sh/users/947499) | Ha sviluppato osu!Rank e osu!Post |

## 2016

## Gennaio

*Per il post, vedi: [Community Contributor Badges (January 2016)](https://osu.ppy.sh/home/news/2016-01-09-community-contributor-badges-january-2016)*

| Utente | Contributi |
| :-- | :-- |
| ::{ flag=US }:: [ztrot](https://osu.ppy.sh/users/6347) | Ha fondato la [osu!academy](/wiki/Community/Video_series/osu!academy) |
| ::{ flag=CA }:: [Tasha](https://osu.ppy.sh/users/1031958) | Collaboratore di osu!weekly e delle notizie. |
| ::{ flag=BG }:: [Flanster](https://osu.ppy.sh/users/447818) | Straordinaria presenza come moderatore (Più di 10000 silence). |
| ::{ flag=PH }:: [Nathanael](https://osu.ppy.sh/users/2295078) | Straordinaria presenza come moderatore (Più di 10000 silence). |
| ::{ flag=SE }:: [Saten](https://osu.ppy.sh/users/444506) | Straordinaria presenza come moderatore e impegno nel gestire la comunità (Più di 30000 silence). |

> | ::{ flag=DE }:: [MoonShade](https://osu.ppy.sh/users/273649) | Ha sviluppato utities rivoluzionarie per fare le storyboard (SGL) |

Reviewer: ```suggestion
| ::{ flag=DE }:: [MoonShade](https://osu.ppy.sh/users/273649) | Ha sviluppato strumenti rivoluzionari per creare le storyboard (SGL) |
```

| ::{ flag=PL }:: [Hatsu](https://osu.ppy.sh/users/322480) | Mikuia.tv - sviluppo di bot per twitch |

> | ::{ flag=DE }:: [Tillerino](https://osu.ppy.sh/users/2070907) | Sviluppo del bot che raccomanda beatmap |

Reviewer: ```suggestion
| ::{ flag=DE }:: [Tillerino](https://osu.ppy.sh/users/2070907) | Sviluppo del bot che consiglia beatmap |
```

### Marzo

*Per il post, vedi: [osu!weekly #53](https://osu.ppy.sh/home/news/2016-03-22-osuweekly-53)*

| Utente | Contributi |
| :-- | :-- |
| ::{ flag=US }:: [Charles445](https://osu.ppy.sh/users/85000) | Anni di sostegno e miglioramento della qualità del mapping, fornendo assistenza in molte occasioni per aiutare a risolvere vari problemi della comunità. |

### Novembre

*Per il post, vedi: [Recognising the Best of the Best](https://osu.ppy.sh/home/news/2016-11-02-recognising-the-best-of-the-best)*

| Utente | Contributi |
| :-- | :-- |

> | ::{ flag=US }:: [pishifat](https://osu.ppy.sh/users/3178418) | Performance esemplare nella costruzione e insegnamento sulle beatmaps |

Reviewer: ```suggestion
| ::{ flag=US }:: [pishifat](https://osu.ppy.sh/users/3178418) | Performance esemplare per la creazione e l'insegnamento delle beatmaps |
```

| ::{ flag=DE }:: [Okoratu](https://osu.ppy.sh/users/1623405) | Eccezionale leadership negli affari di gestione delle beatmaps |
| ::{ flag=HK }:: [IamKwaN](https://osu.ppy.sh/users/1856463) | Eccezionale presenza come moderatore e contributi degni di nota |

> | ::{ flag=PL }:: [Marcin](https://osu.ppy.sh/users/722665) | Eccezionale contributi to negli affari della comunità, organizzazione e moderazione |

Reviewer: ```suggestion
| ::{ flag=PL }:: [Marcin](https://osu.ppy.sh/users/722665) | Eccezionale contributi alla community, in particolare sull'organizzazione e moderazione |
```

| ::{ flag=PL }:: [LiquidPL](https://osu.ppy.sh/users/5044384) | Eccezionale contributo allo sviluppo (osu!next) |

> | ::{ flag=CA }:: [Nyquill](https://osu.ppy.sh/users/682935) | Eccezionale contributo alla community (osu!weekly) |

Reviewer: ```suggestion
| ::{ flag=CA }:: [Nyquill](https://osu.ppy.sh/users/682935) | Eccezionale contributo alla comunità (osu!weekly) |
```

> | ::{ flag=US }:: [Derekku](https://osu.ppy.sh/users/91341) | Gestione e moderazione durante i primi anni della community |

Reviewer: ```suggestion
| ::{ flag=US }:: [Derekku](https://osu.ppy.sh/users/91341) | Gestione e moderazione durante i primi anni della comunità |
```

> | ::{ flag=ES }:: [Trosk-](https://osu.ppy.sh/users/3469385) | Contributo eccezionale agli affari della community |

Reviewer: ```suggestion
| ::{ flag=ES }:: [Trosk-](https://osu.ppy.sh/users/3469385) | Contributo eccezionale alla comunità |
```

> | ::{ flag=FR }:: [Shiro](https://osu.ppy.sh/users/113005) | Contributo eccezionale alla moderazione, organizzazione e la gestione |

Reviewer: ```suggestion
| ::{ flag=FR }:: [Shiro](https://osu.ppy.sh/users/113005) | Contributo eccezionale nella moderazione, organizzazione e la gestione della comunità |
```

## 2017

### Dicembre

*Per il post, vedi: [Community Contributors: 2017](https://osu.ppy.sh/home/news/2017-12-24-community-contributors-2017)*

| Utente | Contributi |
| :-- | :-- |

> | ::{ flag=DE }:: [OnosakiHito](https://osu.ppy.sh/users/290128) | Lavoro eccezionale nel formare l'iniziale commuinity di osu!taiko, svariati anni di presenza come BAT/QAT |

Reviewer: ```suggestion
| ::{ flag=DE }:: [OnosakiHito](https://osu.ppy.sh/users/290128) | Lavoro eccezionale nel formare l'iniziale comunità di osu!taiko, svariati anni di presenza come BAT/QAT |
```

> | ::{ flag=ES }:: [Deif](https://osu.ppy.sh/users/318565) | Eccezionali contributi alla community di osu!catch, alla rielaborazione dei criteri di classificazione e ai  tornei |

Reviewer: ```suggestion
| ::{ flag=ES }:: [Deif](https://osu.ppy.sh/users/318565) | Eccezionali contributi alla comunità di osu!catch, tra cui la rielaborazione dei criteri di classificazione e tornei |
```

| ::{ flag=CH }:: [Irreversible](https://osu.ppy.sh/users/1287964) | Contributi dedicati ai BAT/QAT nel corso di molteplici anni |
| ::{ flag=DE }:: [Nwolf](https://osu.ppy.sh/users/1910766) | Centinaia di ore di statistiche e analisi dei tornei World Cup |
| ::{ flag=GB }:: [Yazzehh](https://osu.ppy.sh/users/7068973) | Eccezionale presenza come arbitro in dozzine di tornei della community |
| ::{ flag=CA }:: [Evrien](https://osu.ppy.sh/users/791660) | Eccezionale performance nel casting, commentario, e resoconti/recap degli eventi |
| ::{ flag=DE }:: [Tom94](https://osu.ppy.sh/users/1857058) | La mente dietro molti miglioramenti a osu!, dai pp, alla riscrittura dell'engine grafico, star rating, e molto altro! |
| ::{ flag=CA }:: [DrabWeb](https://osu.ppy.sh/users/6946022) | Eccezionali contributi al progetto osu!(lazer) |
| ::{ flag=BY }:: [EVAST](https://osu.ppy.sh/users/8195163) | Eccezionali contributi al progetto osu!(lazer) con 90+ pull request e centinaia di commits |

> | ::{ flag=CN }:: [huoyaoyuan](https://osu.ppy.sh/users/2428732) | Eccezionali contributi al progetto osu!(lazer) project |

Reviewer: ```suggestion
| ::{ flag=CN }:: [huoyaoyuan](https://osu.ppy.sh/users/2428732) | Eccezionali contributi al progetto osu!(lazer) |
```

| ::{ flag=PL }:: [Kamikaze](https://osu.ppy.sh/users/2124783) | Capitano del Progetto Amata (osu!mania) |
| ::{ flag=GB }:: [Pope Gadget](https://osu.ppy.sh/users/2288341) | Capitano del Progetto Amata (osu!mania) |
| ::{ flag=AR }:: [Yuii-](https://osu.ppy.sh/users/2935923) | Eccezionali contributi al Programma Community Mentorship |
| ::{ flag=US }:: [Halfslashed](https://osu.ppy.sh/users/4598899) | Eccezionali contributi al Programma Community Mentorship |
| ::{ flag=DE }:: [Mir](https://osu.ppy.sh/users/8688812) | Eccezionali contributi al Programma Community Mentorship |
| ::{ flag=US }:: [Mun](https://osu.ppy.sh/users/6699165) | Eccezionali contributi al Programma Community Mentorship |
| ::{ flag=FI }:: [J1NX1337](https://osu.ppy.sh/users/3971179) | Eccezionali contributi al Programma Community Mentorship |
| ::{ flag=JP }:: [ekr](https://osu.ppy.sh/users/4497706) | Eccezionali contributi al progetto osu!(lazer) |

## 2020

### Febbraio

*Per il post, vedi: [Community Contributors: 2019](https://osu.ppy.sh/home/news/2020-02-07-community-contributors-2019)*

| Utente | Contributi |
| :-- | :-- |

> | ::{ flag=CA }:: [VINXIS](https://osu.ppy.sh/users/4323406) | Eccezionali contributi agli affari della comunità, eventi e tornei |

Reviewer: ```suggestion
| ::{ flag=CA }:: [VINXIS](https://osu.ppy.sh/users/4323406) | Eccezionali contributi alla comunità, eventi e tornei |
```

| ::{ flag=SG }:: [hehe](https://osu.ppy.sh/users/2123087) | Eccezionali contributi alla scena mapping, eventi e tornei |
| ::{ flag=US }:: [Noffy](https://osu.ppy.sh/users/1541323) | Eccezionali contributi alle scene mapping, modding e metadata |
| ::{ flag=SG }:: [Shoegazer](https://osu.ppy.sh/users/2520707) | Eccezionali contributi alla modalità osu!mania |
| ::{ flag=GB }:: [JBHyperion](https://osu.ppy.sh/users/4879508) | Eccezionali contributi alla modalità osu!catch, e la sua gestione |

> | ::{ flag=GB }:: [-Mo-](https://osu.ppy.sh/users/2202163) | Eccezionali contributi agli affati sulla gestione e leadership |

Reviewer: ```suggestion
| ::{ flag=GB }:: [-Mo-](https://osu.ppy.sh/users/2202163) | Eccezionali contributi sulla gestione e leadership |
```

| ::{ flag=US }:: [Chaos](https://osu.ppy.sh/users/2628870) | Eccezionali contributi al Global Moderation Team |
| ::{ flag=BE }:: [yaspo](https://osu.ppy.sh/users/4945926) | Eccezionali contributi al progetto Mentorship |
| ::{ flag=CL }:: [Uberzolik](https://osu.ppy.sh/users/1314547) | Eccezionali contributi al progetto Mentorship |
| ::{ flag=SE }:: [PuffBuck](https://osu.ppy.sh/users/4234525) | Eccezionali contributi alla moderazione e organizzazione della World Cup |
| ::{ flag=GB }:: [Doomsday](https://osu.ppy.sh/users/18983) | Eccezionali, costanti contributi alla comunità di osu in generale |
| ::{ flag=AT }:: [Omgforz](https://osu.ppy.sh/users/578943) | Eccezionali contributi alla osu! World Cup |
| ::{ flag=AU }:: [Kano](https://osu.ppy.sh/users/3036203) | Eccezionali contributi alla osu! World Cup |
| ::{ flag=US }:: [Halogen-](https://osu.ppy.sh/users/169992) | Eccezionali contributi alla scena torneistica di osu!mania |

> | ::{ flag=DE }:: [Junihuhn](https://osu.ppy.sh/users/4182339) | Eccezionali contributi alla osu! World Cup & e tornei |

Reviewer: ```suggestion
| ::{ flag=DE }:: [Junihuhn](https://osu.ppy.sh/users/4182339) | Eccezionali contributi alla osu! World Cup e tornei |
```

| ::{ flag=NL }:: [Sartan](https://osu.ppy.sh/users/4100941) | Eccezionali contributi alla scena torneistica di osu!catch |

> | ::{ flag=RU }:: [Kobold84](https://osu.ppy.sh/users/3227533) | Eccezionali contributi to community moderation |

Reviewer: ```suggestion
| ::{ flag=RU }:: [Kobold84](https://osu.ppy.sh/users/3227533) | Eccezionali contributi nella moderazione della comunità |
```

| ::{ flag=US }:: [Death](https://osu.ppy.sh/users/3242450) | Eccezionali, costanti contributi al supporto tecnico e assistenza |
| ::{ flag=US }:: [Dntm8kmeeatu](https://osu.ppy.sh/users/5428812) | Eccezionali, costanti contributi al supporto tecnico e assistenza |
| ::{ flag=CL }:: [Milan-](https://osu.ppy.sh/users/1052994) | Eccezionali contributi alla Mappers' Guild e Beatmap Nominators |

> | ::{ flag=US }:: [Joehu](https://osu.ppy.sh/users/8549835) | Eccezionali contributi ai progetti opensource di osu! |

Reviewer: ```suggestion
| ::{ flag=US }:: [Joehu](https://osu.ppy.sh/users/8549835) | Eccezionali contributi ai progetti open source di osu! |
```

## 2021

### Marzo

*Per il post, vedi: [Community Contributors: 2020](https://osu.ppy.sh/home/news/2021-03-19-community-contributors-2020)*

| Utente | Contributi |
| :-- | :-- |
| ::{ flag=DE }:: [hallowatcher](https://osu.ppy.sh/users/1874761) | Eccezionali contributi agli eventi della comunità e sviluppo |
| ::{ flag=GB }:: [mangomizer](https://osu.ppy.sh/users/1893718) | Eccezionali contributi ai World Cup ed eventi della comunità |
| ::{ flag=DE }:: [Lasse](https://osu.ppy.sh/users/896613) | Eccezionali contributi alla scena mapping e modding |

> | ::{ flag=PL }:: [spaceman_atlas](https://osu.ppy.sh/users/3035836) | Eccezionali contributi allo sviluppo di osu! development attraverso molti progetti |

Reviewer: ```suggestion
| ::{ flag=PL }:: [spaceman_atlas](https://osu.ppy.sh/users/3035836) | Eccezionali contributi allo sviluppo di osu! attraverso molti progetti |
```

| ::{ flag=US }:: [I Must Decrease](https://osu.ppy.sh/users/2773526) | Eccezionali contributi al mantenimento e lo sviluppo dei punteggi |
| ::{ flag=US }:: [this1neguy](https://osu.ppy.sh/users/1797189) | Eccezionali contributi ai World Cup e alla scena torneistica della comunità |

## 2022

### Giugno

*Per il post, vedi: [Community Contributors: 2021](https://osu.ppy.sh/home/news/2022-06-30-community-contributors-2021)*

| Utente | Contributi |
| :-- | :-- |
| ::{ flag=FR }:: [Kasumi-sama](https://osu.ppy.sh/users/6177263) | Eccezionali contributi alla scena torneistica della community di osu!taiko |
| ::{ flag=SA }:: [frenzibyte](https://osu.ppy.sh/users/14210502) | Eccezionali contributi allo sviluppo di osu!(lazer) |
| ::{ flag=HR }:: [Susko3](https://osu.ppy.sh/users/18945305) | Eccezionali contributi allo sviluppo di osu!(lazer) e osu!framework |
| ::{ flag=RU }:: [StanR](https://osu.ppy.sh/users/7217455) | Eccezionali contributi allo sviluppo e mantenimento dei punti performance di osu! |
| ::{ flag=GB }:: [Apo11o](https://osu.ppy.sh/users/9558549) | Eccezionali contributi allo sviluppo dei punti performance di osu! |

> | ::{ flag=AU }:: [MBmasher](https://osu.ppy.sh/users/4498616) | Eccezionali contributi to sviluppo dei punti performance di osu! e rivitalizzazione della mod Flashlight |

Reviewer: ```suggestion
| ::{ flag=AU }:: [MBmasher](https://osu.ppy.sh/users/4498616) | Eccezionali contributi allo sviluppo dei punti performance di osu! e ha dato nuova vita alla mod Flashlight |
```

| ::{ flag=SE }:: [Walavouchey](https://osu.ppy.sh/users/5773079) | Eccezionali contributi al progetto osu! wiki |
| ::{ flag=ID }:: [Niva](https://osu.ppy.sh/users/197805) | Eccezionali contributi al progetto osu! wiki |
| ::{ flag=NZ }:: [Technocoder](https://osu.ppy.sh/users/10338558) | Eccezionali contributi al supporto tecnico per macOS  |
| ::{ flag=LT }:: [huu](https://osu.ppy.sh/users/6044237) | Eccezionali contributi alla gestione e organizzazione del Progetto Amata |

> | ::{ flag=NL }:: [OliBomby](https://osu.ppy.sh/users/6573093) | Eccezionali contributi alla scena mapping di osu! attraverso sviluppo di utility |

Reviewer: ```suggestion
| ::{ flag=NL }:: [OliBomby](https://osu.ppy.sh/users/6573093) | Eccezionali contributi alla scena mapping di osu! attraverso sviluppo di strumenti |
```

> | ::{ flag=TR }:: [frukoyurdakul](https://osu.ppy.sh/users/7612550) | Eccezionali contributi alla scena mapping di osu!taiko attraverso sviluppo di utility |

Reviewer: ```suggestion
| ::{ flag=TR }:: [frukoyurdakul](https://osu.ppy.sh/users/7612550) | Eccezionali contributi alla scena mapping di osu!taiko attraverso sviluppo di strumenti |
```
